core/controlplane/config/config_test.go

```diff
@@ -1503,10 +1503,10 @@ func TestInvalidKubernetesVersion(t *testing.T) {
 kubernetesVersion: v1.x.3
 `,
 		`
-kubernetesVersion: v1.10.4yes
+kubernetesVersion: v1.10.5yes
```

Comment:
Not much need to update these tests BTW - it's just checking invalid versions error out.

Reply:
Makes sense. I'll leave it as-is because our `contrib/bump-version` script automatically catches it up. But thanks for pointing it out!